e2e-tests/auto-tuning/run

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+cluster="auto-tuning"
+
+
+create_namespace $namespace
+deploy_operator
+
+spinup_pxc "$cluster" "$test_dir/conf/$cluster.yml"
+
+MYSQL_PASSWORD=$(kubectl_bin get secret my-cluster-secrets -o jsonpath='{.data.root}' | base64 -d)
+AUTO_INNODB_SIZE=$(run_mysql \
+                        'SELECT @@innodb_buffer_pool_size;' \
+                        "-h $cluster-pxc -uroot -proot_password")
+AUTO_CONNECTIONS=$(run_mysql \
+                        'SELECT @@max_connections;' \
+                        "-h $cluster-pxc -uroot -proot_password")
+
+kubectl_bin patch pxc "${cluster}" --type merge -p'{"spec": {"pxc": {"configuration": "[mysqld]\ninnodb_buffer_pool_size=150000000\nmax_connections=200\n"}}}'
+
+
+wait_cluster_consistency "${cluster}" 3
+sleep 20
+wait_cluster_consistency "${cluster}" 3
+sleep 20
+wait_cluster_consistency "${cluster}" 3
+sleep 20
+
+INNODB_SIZE=$(run_mysql \
+                        'SELECT @@innodb_buffer_pool_size;' \
+                        "-h $cluster-pxc -uroot -proot_password")
+CONNECTIONS=$(run_mysql \
+                        'SELECT @@max_connections;' \
+                        "-h $cluster-pxc -uroot -proot_password")
+
+if [[ ${AUTO_INNODB_SIZE} == ${INNODB_SIZE} ]]; then
```

Comment:
according to PR logic
AUTO_INNODB_SIZE should be equal to `75000000` but INNODB_SIZE should be equal `150000000`
AUTO_CONNECTIONS = `7`, CONNECTIONS = `200`